src/blocks/recommendations/components/parsely-recommendations-block-controls.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { BlockControls } from '@wordpress/block-editor';
+import { Dashicon, ToolbarGroup } from '@wordpress/components';
+
+const ParselyRecommendationsBlockControls = ( { attributes: { layoutstyle }, setAttributes } ) => {
```

Comment:
Let's consider removing the "Layout Style" toggle here and in the "Inspector Controls" (if it's pulled out, we should remove support for the `layoutstyle` attribute throughout).

Reply:
@jblz could you remind us of the reasoning for this, and what would be the desired result/advantage of doing so?

Reply:
@acicovic Just to simplify the MVP. The least complex rendering would be as simple an unordered-list as possible (minimal CSS and intermediate container nesting added by our block). This would push responsibility to the theme / site implementer to ensure the design elements flow with the rest of the site and would not lock us into supporting a more complex layout going forward until we're confident it's a real winner.

Reply:
Let's remove it. We should also create an issue to track it and implement it later, perhaps we can paste some code in that issue to pick it up easily.

Reply:
This has been removed in ffefdce and being tracked in #708, so marking this as resolved.